keras_nlp/models/bert/bert_presets.py

```diff
@@ -238,6 +238,7 @@
                 },
             },
             "num_classes": 2,
+            "dropout": 0.1,
```

Comment:
We should probably retrain with this change right (unless it makes no difference?). Should we open a follow up?